#goal definition
uint8   action_id
uint8   distance
uint8   angle
---
#result definition
uint8   action_id
string  result
---
#feedback
uint8   action_id
string  status